mypy.ini

```diff
@@ -31,10 +34,14 @@ disable_error_code = attr-defined
 disable_error_code = import-not-found
 
 # - distutils._modified has different errors on Python 3.8 [import-untyped], on Python 3.9+ [import-not-found]
-# - All jaraco modules are still untyped
-# - _validate_project sometimes complains about trove_classifiers (#4296)
-# - wheel appears to be untyped
-[mypy-distutils._modified,jaraco.*,trove_classifiers,wheel.*]
+# - The following are not marked as py.typed:
```

Comment:
Comment for cross-linking:
- jaraco.envs: https://github.com/jaraco/jaraco.envs/issues/7
- jaraco.packaging: https://github.com/jaraco/jaraco.packaging/issues/20
- jaraco.path: https://github.com/jaraco/jaraco.path/issues/2
- jaraco.test: https://github.com/jaraco/jaraco.test/issues/7
- jaraco.text: https://github.com/jaraco/jaraco.text/issues/17
- wheel: does not intend on exposing a programatic API https://github.com/pypa/wheel/pull/610#issuecomment-2081687671